app/models/google_calendar/event_finder.rb

```diff
@@ -20,8 +20,10 @@ def confirmed_events(time_interval)
       all_google_events.select { |event| confirmed_ids.include?(event.id) }
     end
 
-    def by_room(time_interval, conference_room_ids)
-      list_events(time_interval, rooms(conference_room_ids))
+    def by_room(time_interval, conference_room_ids, with_confirmation = false)
```

Comment:
I believe that using boolean flag to control method's behaviour is not recommended. Probably it's better to create another method that specializes in particular task